ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -67,15 +67,24 @@ chain:
         export KUBECONFIG=${SHARED_DIR}/kubeconfig
       fi
 
-      # In order for openshift-tests to pull external binary images from the
-      # payload, we need access enabled to the images on the build farm. In
-      # order to do that, we need to unset the KUBECONFIG so we talk to the
-      # build farm, not the cluster under test.
-      echo "Granting access for image pulling from the build farm..."
-      KUBECONFIG_BAK=$KUBECONFIG
-      unset KUBECONFIG
-      oc adm policy add-role-to-group system:image-puller system:unauthenticated --namespace "${NAMESPACE}"
-      export KUBECONFIG=$KUBECONFIG_BAK
+      if [[ $MULTISTAGE_PARAM_OVERRIDE_TEST_EXTRACT == "true" ]]; then
+        RELEASE_IMAGE_LATEST="${HOSTEDCLUSTER_RELEASE_IMAGE_LATEST}"
+        oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
+        mkdir /tmp/tests-cli
+        oc image extract "$(oc adm release info --image-for=tests "${HOSTEDCLUSTER_RELEASE_IMAGE_LATEST}")" --path /usr/bin/openshift-tests:/tmp/tests-cli --registry-config=/tmp/.dockerconfigjson
```

Comment:
I checked OCP 4.18 nightly payload, the tests image is in it, we don't need to extract new one `openshift-tests`.
```
$ oc adm release info --commits registry.ci.openshift.org/ocp/release:4.18.0-0.nightly-2025-03-06-032341 | grep origin
  tests                                          https://github.com/openshift/origin                                         c89253a2124ee5f90bfbf92b05085ae596e85a23

```